tests/openshift/pipelines/Postman-Test-JenkinsFile.groovy

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env groovy
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
Inconsistent comment style with the rest of the other headers (the others use the multiline /* */ style, but this one is doing it per line).